src/ScalarMessaging.js

```diff
@@ -564,12 +564,27 @@ const onMessage = function(event) {
     });
 };
 
+let listenerCount = 0;
```

Comment:
Be very careful with module level variables - if you ever call these functions from both react-sdk and vector-web you're going to become very confused (you'll get 2 instances of the module and therefore 2 separate variables).